tests/test_seasonal_water_yield_regression.py

```diff
@@ -451,7 +452,7 @@ def test_aggregate_vector_preexists(self):
         l_path = os.path.join(self.workspace_dir, 'L.tif')
         make_recharge_raster(l_path)
         aggregate_vector_path = os.path.join(self.workspace_dir,
-                                             'aggregated_results.shp')
+                                             AGGREGATED_RESULTS_FILENAME)

```

Comment:
Very interesting, and very DRY!  In this case, I feel like the benefit of having the filename written in-place (even if it is a bit redundant) might outweigh the benefits of having the filename only defined in one place.  There are definitely some cases where being a little more explicit will end up making things easier for us to maintain in the long run and I wonder if this indirection is an example of that ... but I'd love to hear what you think!

And by the way, one of the things that I do like about this approach is that even though there is a bit of indirection, we almost don't lose the meaning here because of the variable name you've chosen ... very nice and explicit and human-readable.

Reply:
Thanks for the feedback, I wasn't sure which way to go on this. I suppose it's really unlikely that this filename will be changing much in the future, so the benefit of having it defined in one place doesn't matter.